users.go

```diff
@@ -4,6 +4,7 @@ import "gopkg.in/errgo.v1"
 
 type User struct {
 	ID                  string          `json:"id"`
+	UUID                string          `json:"uuid"`
```

Comment:
Again this question of 'should the `id` be the `uuid`, I think it should be', duplicated Identifier is wrong to me.

Reply:
Yep but i'm unable to delete it without breaking retro-compatibility.

Reply:
How so? I don't get it, can you expand your answer?

Reply:
If i remove the ID field it might break some dependency since they can use it their code. Removing it from the struct will prevent other codes from compiling.

Reply:
It's the same reason why we kept the two fields in the API

Reply:
It's now what I mean, actually I think the `UUID` field should not exist.

If we change the structure of the `ID` by a `uuid`, it's at our discretion.

Reply:
Where we ensure retrocompatibility is that if a client sends an *old* ID, it will still work